README.adoc

```diff
@@ -184,33 +206,39 @@ include::finish/src/main/java/io/openliberty/guides/jpaguide/dao/EventDao.java[t
 ----
 
 In order to use the entity manager at runtime, it must be injected into our EJB component through the
-`@PersistenceContext` annotation. Since the entity manager is container-managed, the EJB container
-will ensure that the methods are executed within an active transaction which is why it is not necessary
-to explicitly begin, commit, or rollback transactions. At the end of an EJB operation, the persistence 
-context will be notified that the transaction is committing and the persistence provider, EclipseLink,
-will flush the changes to the database.
+`@PersistenceContext` annotation. Since the stateless session bean is configured to manage the global
+transactions, the EJB container will ensure that the required methods are executed within the boundaries
+of an active global transaction, which is why it is not necessary to explicitly begin, commit or rollback
+transactions. At the end of a session bean method invocation, the persistence context will be notified
```

Comment:
That's effectively true for the guide's application that I've seen so far.  But generally speaking, the business method that began the transaction in its preInvoke phase is the same business method that will commit/rollback the transaction in its postInvoke phase.

Things get more complicated when you have session beans business methods calling the business methods of other session beans, because you have the elements of transaction propagation (or transaction suspension) and persistence context propagation.